packages/e2e-tests/cypress/integration/eligibility/householder-planning/enforcement-notice/enforcement-notice-steps.js

```diff
@@ -0,0 +1,58 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+import { goToAppealsPage } from '../../../../support/common/go-to-page/goToAppealsPage';
+import { verifyPageHeading } from '../../../../support/common/verify-page-heading';
+import { verifyPageTitle } from '../../../../support/common/verify-page-title';
+import {
+  getEnforcementNoticeErrorMessage,
+  getEnforcementNoticeNo, getEnforcementNoticeYes,
+} from '../../../../support/eligibility/page-objects/enforcement-notice-po';
+import { getErrorMessageSummary } from '../../../../support/common-page-objects/common-po';
+import { verifyErrorMessage } from '../../../../support/common/verify-error-message';
+import { getBackLink } from '../../../../support/common-page-objects/common-po';
+import { getContinueButton } from '../../../../support/householder-planning/appeals-service/page-objects/common-po';
+const pageHeading = 'Have you received an enforcement notice?';
+const pageTitle = 'Have you received an enforcement notice? - Before you start - Appeal a planning decision - GOV.UK';
+const url = `before-you-start/enforcement-notice-householder`;
+
+Given('appellant is on the enforcement notice page for householder planning', () => {
+  goToAppealsPage(url, { headers: { 'Referer': `${Cypress.env('APPEALS_BASE_URL')}/before-you-start/decision-date-householder` } });
```

Comment:
Why do we have referer here?

Reply:
To know from which page the user is coming to enforcement notice page